ArcGISRuntimeSDKQt_QMLSamples/DisplayInformation/ShowCallout/README.md

```diff
@@ -1,15 +1,33 @@
 # Show callout
 
-This sample demonstrates how to show location coordinates on a `Map` using a `Callout`.
+Show a callout with the latitude and longitude of user-tapped points.
 
 ![](screenshot.png)
 
+## Use case
+
+Callouts are used to display temporary detail content on a map. You can display text and arbitrary UI controls in callouts.
+
+## How to use the sample
+
+Tap anywhere on the map. A callout showing the WGS84 coordinates for the tapped point will appear.
+
 ## How it works
-A `Callout` is created, and its `calloutData` property is bound to a `MapView`'s calloutData. Using the `onMouseClicked` signal handler, the calloutData's `detail` property is edited to display the mouseclick's coordinates.
-
-## Features
-- Map
-- MapView
-- Callout
-- Viewpoint
-- SpatialReference
+
+1. Listen for `mouseClicked` signal on the map view.
+2. When the user taps, get the tapped location using the mouseEvent x and y coordinates, `MapQuickView.screenToLocation(mouse.x, mouse.y)`.
+3. Create a string to display the coordinates; note that latitude and longitude in WGS84 map to the Y and X coordinates.
+4. Create a new callout definition using a title and the coordinate string.
+5. Display the callout by setting the`location` property of the `CalloutData` and calling `showCallout`.
+
+## Relevant API
+
+* CalloutDefinition
+* GeometryEngine.project
+* GeoViewTappedEventArgs
+* MapView.geoViewTapped
+* MapView.showCallout
```

Comment:
We don't have a MapView.showCallout. The Callout itself has showCallout - https://developers.arcgis.com/qt/latest/toolkit/api-reference/qml-callout.html#showCallout-method